docs/index.md

```diff
@@ -39,28 +39,27 @@ Each front end tool must accept the Mother Tongues data format. The schema for t
 Making a Dictionary
 -------------------
 
-To make a dictionary from your data, you need to do the following three
-things:
+Mother Tongues Dictionaries are intentionally designed with different communities and users in mind. To that end, we offer different paths you can take to create your dictionary.
 
-1.  Install mothertongues, see [installation](install.md).
-2.  Write a valid Mother Tongues Language configuration file. See
-    [guides](guides/index.md)
-3.  Write valid Mother Tongues data resource configuration files. See
-    [guides](guides/index.md)
+### Option 1: No-Code Option (Fastest)
+If any of the following describe you, then the no-code path may be right for you:
 
-Then, build with the command line:
+- I want my dictionary running as quickly as possible with the least lift
+- I want to try out-of-the-box MTD "as-is" (in other words, you don't need to make major code customizations right now)
```

Comment:
maybe add "I'm not as comfortable with coding" or something similar?

Reply:
added